### PR TITLE
[Issue #2976] Run analytics migrations like the API does them

### LINKED ISSUE #2976: Run database migrations for the analytics application automatically
### Summary

//

### Acceptance criteria

//

### PR DESCRIPTION
**Pull request by coilysiren**
## Summary

Fixes #2976

## Changes proposed

- Adds "db migrate" poetry command
- Renames "db init" to "db migrate"
- Removes `init-etldb` cronjob since we are now running that same logic on merge
- Starts running migrations on merge (`infra/analytics/app-config/outputs.tf` activates this)

## Testing

https://github.com/HHS/simpler-grants-gov/actions/runs/11960219381/job/33346703132#step:5:256

image

### CODE REVIEW

**Review by mdragon — APPROVED** [review]
Looks good.

File: analytics/src/analytics/integrations/etldb/main.py [hunk 1]
```diff
@@ -18,7 +18,7 @@
 VERBOSE = False
 
 
-def initialize_database() -> None:
+def db_migrate() -> None:
```
Comment (DavidDudas-Intuitial) [inline analytics/src/analytics/integrations/etldb/main.py:21]: nit: please change this to `migrate_database`

File: analytics/src/analytics/cli.py [hunk 2]
```diff
@@ -264,12 +264,12 @@ def export_json_to_database(delivery_file: Annotated[str, ISSUE_FILE_ARG]) -> No
 # ===========================================================
 
 
-@etl_app.command(name="initialize_database")
-@ecs_background_task("initialize_database")
-def initialize_database() -> None:
+@etl_app.command(name="db_migrate")
+@ecs_background_task("db_migrate")
+def db_migrate() -> None:
```
Comment (DavidDudas-Intuitial) [inline analytics/src/analytics/cli.py:269]: nit: please change this to `migrate_database`

**Review by DavidDudas-Intuitial — CHANGES_REQUESTED** [review]

File: analytics/src/analytics/integrations/etldb/main.py (hunk 1 above)
Comment (coilysiren) [reply analytics/src/analytics/integrations/etldb/main.py]: https://github.com/HHS/simpler-grants-gov/pull/2978/commits/84ec4d70482be42b21e9717ccc5d58bc80b59346 ✅

**Review by DavidDudas-Intuitial — APPROVED** [review]

**Pull request merged**